src/clj/rems/api/resources.clj

```diff
@@ -72,9 +72,11 @@
     (GET "/" []
       :summary "Get resources"
       :roles #{:owner}
-      :query-params [{active :- (describe s/Bool "filter active or inactive resources") nil}]
+      :query-params [{active :- (describe s/Bool "filter active or inactive resources") nil}
+                     {archived :- (describe s/Bool "whether to include archived resources") false}]
       :return Resources
-      (ok (get-resources (when active {:active active}))))
+      (ok (get-resources (merge (when active {:active active})
+                                (when-not archived {:archived false})))))
```

Comment:
Since you are giving archived a default value that is either `true` or `false` could you just not use the when tricks here?

Reply:
`{:archived true}` would return *only* archived resources, but we want to return *both* archived and non-archived resources.